### PR TITLE
Small paper edits

### PR DESCRIPTION
**Pull request by jjc2718**
The paper looks good! Just a few suggestions for clarity, totally up to you whether to take or leave them. Overall I think it's coming together nicely!

### CODE REVIEW

**Comment by jjc2718** [comment]
@ben-heil apparently I can't add reviewers, but if you want to take a look that would be great!

File: content/03.results.md [hunk 1]
```diff
@@ -31,20 +31,20 @@ We used the same experimental setup as in our other binary prediction tasks to t
 In this setting we found that while the models all performed similarly, the nonlinear models tended to have a slight edge over the linear one.
 
 ![
-Performance of models across four classification tasks
+Performance of models across four classification tasks. In each panel, confidence intervals show mean and standard error across 5 folds of studywise cross-validation.
```
Comment (ben-heil) [inline content/03.results.md:34]: Is this true? I'm using the default `geom_smooth` function in plotnine which is loess curve + 95% confidence interval. I think that's a local regression line and it's confidence, not the true mean and SE, but I could be wrong

**Review by ben-heil — APPROVED** [review]
Thanks! I'm going to merge this then tweak the sections on balanced accuracy and the loess curve description

File: content/03.results.md (hunk 1 above)
Comment (jjc2718) [reply content/03.results.md:34]: Ahh, yeah, I think you're right!

**Pull request merged**